app/assets/stylesheets/application.css.scss

```diff
@@ -63,6 +63,27 @@ footer {
   padding: 10px;
 }
 
+.splash-title {
+  font-family: 'Miltonian Tattoo', cursive;
+  margin: 0;
+  text-align: center;
+}
+
+.splash-body {
+  font-family: 'Miltonian Tattoo', cursive;
+  text-align: center;
+  a {
+
+    color: #000000;
```

Comment:
Color literals like `#000000` should only be used in variable declarations; they should be referred to via variable everywhere else.